### ISSUE TITLE
Add events to day without start and end time

### ISSUE THREAD

**Issue by meron1122** [Serhioromano/bootstrap-calendar#517]
Is posible to ignore the start end end value?

**Comment by eleazan** [comment]
No, you can't...

But you can do a event for "all day", adding the times manually, setting it before time_start (default, 06:00 ) and after time_end (default, 22:00 )

**Comment by meron1122** [comment]
Thank you for response.
I can have for example 10 all day events?
I want only place event in day, nothing more.

**Comment by eleazan** [comment]
The same day?

At month view, you can look it fine... but if you change to the day view,
you probably look so bad rendered... try! ;)

**Comment by meron1122** [comment]
Yup, on moth view, i unbind day view, in my proporse is usless :)
